SampleDanger/ViewController.swift

```diff
@@ -649,6 +649,647 @@ class ViewController: UIViewController {
                                          
     // Do any additional setup after loading the view.
   }
+  
+  func check() {
```

Comment:
<!--
  1 failure:  Function body sho...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 653;
-->
- :no_entry_sign: Function body should span 40 lines or less excluding comments and whitespace: currently spans 633 lines (`function_body_length`)